src/engine/sparqlExpressions/SparqlExpressionValueGetters.cpp

```diff
@@ -90,6 +90,15 @@ std::optional<std::string> StringValueGetter::operator()(
   }
 }
 
+// ____________________________________________________________________________
+std::optional<LiteralOrIri> LiteralOrIriValueGetter::operator()(
+    Id id, const EvaluationContext* context) const {
+  // true means that immediately returns nullopt for everything that is not a
+  // literal
```

Comment:
The comment is out of sync. it says `true means` but the only thing I see in the call is a `false`.